app/assets/javascripts/student_profile/services_list.jsx

```diff
@@ -251,11 +251,19 @@ import _ from 'lodash';
       const isConfirming = (this.state.discontinuingServiceId === service.id);
       const isHovering = (this.state.hoveringDiscontinueServiceId === service.id);
       const isPending = (this.props.discontinueServiceRequests[service.id] === 'pending');
+      var Text = 'Confirm';
```

Comment:
Since we're using ES6, let's prefer `const` or `let` over `var` because `const` or `let` will say more about our intention for the variable.

Reply:
Let's also use camelcase names for variables and have them start with a lowercase letter, like `text` instead of `Text`.